README.md

```diff
@@ -1,2 +1,246 @@
+| `livenessProbe.successThreshold`     | Success threshold for liveness probe                                                                          | `1`                                  |
+| `readinessProbe.enabled`             | Use this parameter to enable the readiness probe                                                              | `true`                               |
+| `readinessProbe.initialDelaySeconds` | Initial delay seconds for readiness probe                                                                     | `10`                                 |
+| `readinessProbe.periodSeconds`       | Period seconds for readiness probe                                                                            | `60`                                 |
+| `readinessProbe.timeoutSeconds`      | Timeout second for readiness probe                                                                            | `5`                                  |
+| `readinessProbe.failureThreshold`    | Failure threshold for readiness probe                                                                         | `3`                                  |
+| `readinessProbe.successThreshold`    | Success threshold for readiness probe                                                                         | `1`                                  |
+| `startupProbe.enabled`               | Parameter to enable startup probe                                                                             | `true`                               |
+| `startupProbe.initialDelaySeconds`   | Initial delay seconds for startup probe                                                                       | `10`                                 |
+| `startupProbe.periodSeconds`         | Period seconds for startup probe                                                                              | `20`                                 |
+| `startupProbe.timeoutSeconds`        | Timeout second for startup probe                                                                              | `1`                                  |
+| `startupProbe.failureThreshold`      | Failure threshold for startup probe                                                                           | `30`                                 |
+| `startupProbe.successThreshold`      | Success threshold for startup probe                                                                           | `1`                                  |
+| `persistence.enabled`                | Enable MarkLogic data persistence using Persistence Volume Claim(PVC). If set to false, EmptyDir will be used | `true`                               |
+| `persistence.storageClass`           | Storage class for MarkLogic data volume, leave empty will use the default storage class                       | `""`                                 |
```

Comment:
Storage class for MarkLogic data volume, leave empty to use the default storage class